pkg/crc/machine/vsock.go

```diff
@@ -43,6 +41,29 @@ func isOpened(exposed []types.ExposeRequest, port types.ExposeRequest) bool {
 	return false
 }
 
+func unExposePorts() error {
+	daemonClient, alreadyOpenedPorts, err := getClientAndOpenPorts()
+	if err != nil {
+		return err
+	}
+	for _, port := range alreadyOpenedPorts {
+		if err := daemonClient.NetworkClient.Unexpose(&types.UnexposeRequest{Local: port.Local}); err != nil {
+			return errors.Wrapf(err, "failed to unexpose port %s ", port.Local)
+		}
+	}
+	return nil
+}
+
+func getClientAndOpenPorts() (*daemonclient.Client, []types.ExposeRequest, error) {
+	daemonClient := daemonclient.New()
+	alreadyOpenedPorts, err := daemonClient.NetworkClient.List()
+	if err != nil {
+		logging.Error("Is 'crc daemon' running? Network mode 'vsock' requires 'crc daemon' to be running, run it manually on different terminal/tab")
+		return daemonClient, nil, err
```

Comment:
`return nil, nil ,err`, though I'd prefer to only return `([]types.ExposeRequest, error)`
It might even be simpler to just add a constant for the `logging.Error()` parameter, and to not add a helper for this.